libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_votes_packet_handler.cpp

```diff
@@ -47,7 +176,7 @@ void ExtVotesPacketHandler::sendPbftVotes(const dev::p2p::NodeID &peer_id, std::
     dev::RLPStream s(count);
     for (auto i = index; i < index + count; i++) {
       const auto &v = votes[i];
-      s.appendRaw(v->rlp(true));
+      s.appendRaw(v->rlp(true, true));
```

Comment:
why are you include weight?

Reply:
because the node did not work anymore due to some assert related to empty weight, but I did not spend too much tie thinking about other possible fixes so I can revisit this one